autoload/tnt.wad/brghtmps.lmp

```diff
@@ -0,0 +1,22 @@
+// [crispy] Final Doom textures
```

Comment:
These are meant to be loaded *additionally* to the ones in doom-all.

Reply:
Yes, it works this way. We parse multiple BRGHTMPS lumps, like DEHACKED or UMAPINFO.